pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1266,13 +1259,31 @@ func getDefaultQueue(append bool, defaultJmsQueue string, jmsQueue string) strin
 	return ""
 }
 
-func setPasswords(spec *api.KieAppSpec, isTrialEnv bool) {
+func setPasswords(spec *api.KieAppSpec, status api.KieAppStatus, isTrialEnv bool) {
 	passwords := []*string{
 		&spec.CommonConfig.KeyStorePassword,
 		&spec.CommonConfig.DBPassword,
 		&spec.CommonConfig.AMQPassword,
 		&spec.CommonConfig.AMQClusterPassword,
 	}
+
+	if len(spec.CommonConfig.SecretAdminCredentials) == 0 {
```

Comment:
can you move this if to a new function. It would be better for understanding the code to read the function name then understanding what this if block does.

Reply:
ok